lua/lualine/themes/solarized-osaka.lua

```diff
@@ -1,11 +1,12 @@
 local colors = require("solarized-osaka.colors").setup({ transform = true })
 local config = require("solarized-osaka.config").options
+local util = require("solarized-osaka.util")
 
 local solarized_osaka = {}
 
 solarized_osaka.normal = {
   a = { bg = colors.blue, fg = colors.black },
-  b = { bg = colors.fg, fg = colors.black },
+  b = { bg = util.lighten(colors.bg_statusline, 0.9, "#ffffff"), fg = colors.fg },
```

Comment:
Can you pick a color from the palette instead of manipulating the color? https://github.com/craftzdog/solarized-osaka.nvim/blob/main/lua/solarized-osaka/colors.lua

Reply:
I have just picked some of colors from the palette, but can't really decide which one is the best.